apps/web/public/static/locales/en/common.json

```
@@ -1957,7 +1957,7 @@
   "members_default_location": "Member's default location",
   "members_default_schedule_description": "We will use each members default availability schedule. They will be able to edit or change it.",
   "requires_at_least_one_schedule": "You are required to have at least one schedule",
-  "default_conferencing_bulk_description": "Update the locations for the selected event types",
+  "default_conferencing_bulk_description": "Update the schedules for the selected event types",
```

There are other places where the modal is used for updating locations. Instead pass the description text as prop for the Modal

---

```suggestion
  "default_conferencing_bulk_description": "Update the locations for the selected event types",
  "default_schedules_bulk_description": "Update the schedules for the selected event types",
```

You can then have pass different texts based on what the modal is doing.

---

ok so rather than having a key here, I will remove this and direclty put this text there right? also do I have to manually remove it from all otehr localiztion file?

---

No don't remove the key here, create another key for schedules text. Pass that key when the modal is being used to update schedules. You can pass the other key when updating locations.

---

got it.

---

Also how do I generate it for other languges?

---

You will have to manually do the changes.

---

Just want to confirm. The other place where we are using keyword "location", is it Applist in installed apps sections when we are chagin default?